server/validator.js

```diff
@@ -0,0 +1,28 @@
+const validator = () => {
```

Comment:
This function name is too generic, can we make it more specific???.
From the implementation of the function, what the function seems to be doing is to verify that all the required environment variables are available, so a name like `validateEnvironmentVars` might be better(just a suggestion).

Reply:
The function has been renamed